lib/scala/project-manager/src/main/scala/org/enso/projectmanager/infrastructure/http/ProjectsEndpoint.scala

```diff
@@ -30,23 +30,25 @@ final class ProjectsEndpoint[
     extends Endpoint
     with LazyLogging {
 
-  private val projectRepository =
-    projectRepositoryFactory.getProjectRepository(None)
-
   /** @inheritdoc */
   override def route: Route =
     projectsEndpoint
 
   private val projectsEndpoint = {
     path("projects" / JavaUUID / "enso-project") { projectId =>
-      get {
-        getEnsoProject(projectId)
+      parameters("subdirectory".optional) { directory =>
```

Comment:
it can be a `projectsDirectory` optional parameter. It will be consistent with other methods like `project/open`, `project/create` or `--filesystem-*`

Reply:
Thanks, wasn't aware the GUI deals with full paths